[ADD] inventory: dispatch management system (#11224)
odoo/documentation: content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/dispatch.rst

include:

- *Load building*: Group products for specific carriers, assign those :doc:`batches
  <../picking_methods/batch>` to loading docks, and manage vehicle assignments based on fleet
  capacity. This ensures the right products are packed into the appropriate trucks for delivery.
- :doc:`Fleet management <../../../../hr/fleet>`: Track and manage the capacity of in-house
  delivery vehicles.

Configuration
=============

To use the dispatch management system, the following setup must be completed:

#. Install **Fleet**.
#. Configure vehicle's volume and weight :ref:`capacity
   <inventory/shipping_receiving/vehicle-capacity>`.
#. Enter the :doc:`car model(s) <../../../../hr/fleet/models>` of the vehicles.
#. Enable :ref:`necessary features <inventory/shipping_receiving/inventory-features>` in the
   **Inventory** app.
#. Set up :ref:`vehicle as a delivery method
   <inventory/shipping_receiving/delivery-method-for-carrier>`.
#. Create :ref:`dock locations <inventory/shipping_receiving/docks>`.

.. _inventory/shipping_receiving/vehicle-capacity:

Vehicle capacity
----------------

To configure the vehicle capacity, go to :menuselection:`Fleet app --> Configuration -->

> Categories`.

Author: While this helps users find the button easier, according to @samueljlieber , we won't be including the section title of the dropdowns anymore. So I'll be ignoring this suggestion. Thank you though, Lara :blush: I'll consider this when the drop-downs contain a LOT more items, like the Technical menu.

> Categories`.

Reviewer: Yes, specifically regarding this change in the guidelines 🙂 https://www.odoo.com/documentation/master/contributing/documentation/rst_guidelines.html#menu-selection

Next, click :guilabel:`New` or an existing category. In the blank :guilabel:`Name` field, enter the
type of vehicle, e.g. `Pick-up truck`, `Van`, `Cargo truck`. Then, enter vehicle capacity in the

> :guilabel:`Max Weight` (in kilograms) and :guilabel:`Volume` (in cubic meters).

Author: I'm not sure how to change these units of measure :see_no_evil:

.. seealso::
   :ref:`Vehicle category <fleet/categories>`

.. image:: dispatch/categories.png
   :alt: Vehicle categories with defined weight and volume.

Car model
---------

Configuring a vehicle's car model is required when adding vehicles in Odoo. It is helpful to assign
the Category of the car model to the vehicle category set in the :ref:`Vehicle capacity
<inventory/shipping_receiving/vehicle-capacity>` section. This automatically applies the capacity

> limits from the category to any new vehicles of this car's make and model.

Author: Thank you for elaborating! :D

Locations`.

Choose the desired location, and on the :guilabel:`Location` page, in the :guilabel:`Additional
Information` section, tick the :guilabel:`Is a Dock Location` checkbox.

.. figure:: dispatch/dock-location.png
  :alt: Location configuration.

  Location configuration page with :guilabel:`Is a Dock Location` checkbox ticked.

Build loads
===========

After completing setup, batches of orders can be grouped together to load onto a delivery truck. To
do that, orders must be :ref:`assigned to the carrier
<inventory/shipping_receiving/assign-carrier>`. Then, group delivery orders together to :ref:`create
a batch <inventory/shipping_receiving/create-batch>`. Finally, :ref:`configure the batch form
<inventory/shipping_receiving/batch-form>` correctly.

Since this article is about a specific use case, the details about each picking methods can be
further explored in their dedicated articles.

.. seealso::
   - :doc:`../picking_methods/batch`
   - :doc:`Wave picking <../picking_methods/wave>`
   - :doc:`../picking_methods/cluster`

To group products, go to the :menuselection:`Inventory app --> Operations --> Deliveries`, which

> reveals a list of outgoing deliveries.

Reviewer 2: I agree with this suggestion

> reveals a list of outgoing deliveries.

Author: Hi @ksc-odoo !

According to the [content guidelines](https://www.odoo.com/documentation/16.0/contributing/documentation/content_guidelines.html#lists),  it seems that section names are no longer included when navigating through menu items, and we're now listing only the items themselves. Could you confirm if this is correct?

cc: @samueljlieber

.. _inventory/shipping_receiving/assign-carrier:

Carrier assignment
------------------

Reveal the :guilabel:`Carrier` column if it is not visible by default, by clicking the
:icon:`oi-settings-adjust` :guilabel:`(settings)` icon and ticking the :guilabel:`Carrier` checkbox.

.. tip::
   Other useful columns to enable can be :guilabel:`Zip` code, :guilabel:`Shipping Weight` and
   :guilabel:`Shipping Volume`.

Next, tick the checkbox to far left of the delivery orders intended to go into the batch. After the
deliveries are selected, click into one of the line's :guilabel:`Carrier` fields. In the resulting
drop-down menu, choose the desired vehicle's :ref:`delivery method

> <inventory/shipping_receiving/delivery-method-for-carrier>`. In the resulting
> :guilabel:`Confirmation` pop-up window, click :guilabel:`Confirm`.

Reviewer 3: ```suggestion
<inventory/shipping_receiving/delivery-method-for-carrier> `.  A 
:guilabel:`Confirmation` pop-up window appears, indicating the number of orders being added to the batch. Click :guilabel:`Confirm` and the carrier is updated for all the selected records.
```

.. figure:: dispatch/set-carrier.png
   :alt: Set carrier.

   The delivery method `Truck 1-MER-001` is set as the :guilabel:`Carrier` for two delivery orders.

.. _inventory/shipping_receiving/create-batch:

Create batch
------------

> With the carrier set, add the orders to a batch by having the desired orders checked. Then, click
> the :icon:`fa-cog` :guilabel:`Actions` button and click either :guilabel:`Add to batch` or
> :guilabel:`Add to wave` to group the orders. In the resulting pop-up window, ensure the
> :guilabel:`Add to` field is set :guilabel:`a new [batch/wave] transfer`, and click

Reviewer 3: ```suggestion
With the carrier set, add the desired orders to a batch by ticking the checkbox next to each order. Then, click
the :icon:`fa-cog` :guilabel:`Actions` button and click either :guilabel:`Add to batch` or
:guilabel:`Add to wave` to group the orders. In the resulting pop-up window, ensure the
:guilabel:`Add to` field is set to :guilabel:`a new [batch/wave] transfer`, and click
```
I would add a note directly after the first sentence to explain that the ticked orders that had the carrier applies remain ticked. And that ONLY if they wanted to add more orders, they need to tick those additional boxes (but they should add the carrier). Just a thought- since they would typically be adding what they just set the carrier to. I just noticed they stayed ticked which seemed unusual, so you can expand on that a touch.

Author: Thank you! this is a helpful note to add

- :guilabel:`Operation Type`: from the drop-down menu, select the operation type under which the
  picking is categorized.
- :guilabel:`Scheduled Date`: specifies the date by which the :guilabel:`Responsible` person should
  complete the transfer to the output location.
- :guilabel:`Dock Location`: select the loading location.
- :guilabel:`Vehicle`: select the vehicle, which will auto-fill :guilabel:`Vehicle Category`.
- :guilabel:`Vehicle Category`: configure the vehicle category to determine the vehicle's capacity.
  Items in the batch, with the weight and volume fields configured on the product form, are totaled
  to show how much space is remaining for loading products.

The :guilabel:`Weight` and :guilabel:`Volume` fields display whether the items in the order exceed
the :ref:`capacity set on the vehicle category <inventory/shipping_receiving/vehicle-capacity>`.

.. example::
   The :guilabel:`Volume` bar is grayed out because the capacity has been reached.

   .. image:: dispatch/batch-form.png
      :alt: Show batch form.

Prepare delivery route
~~~~~~~~~~~~~~~~~~~~~~

Additional resources are available to prepare the driver for the delivery. On the top of the batch

> or wave form, click the :guilabel:`Map`. Doing so opens another page, displaying a map. Selecting an

Reviewer 3: ```suggestion
or wave form, click the :guilabel:`Map` button. Doing so opens another page, displaying a map. Selecting an
```

Reviewer 3: Just a note- mine did not open another tab or page- it just went to the map. Might want to double-check and change accordingly.

corner to create a new model. Then, set the relevant :guilabel:`Category` field to the relevant
vehicle category.

.. _inventory/shipping_receiving/inventory-features:

Inventory settings
------------------

Next, go to :menuselection:`Inventory app --> Configuration --> Settings` and enable the required
features for dispatch management.

In the :guilabel:`Operations` section, tick the :guilabel:`Batch, Wave & Cluster Transfers`
checkbox to prepare batches of orders for delivery.

In the :guilabel:`Shipping` section, tick the :guilabel:`Delivery Methods` and :guilabel:`Dispatch
Management System` checkboxes. Doing so allows specific vehicles to be :ref:`set as carriers
<inventory/shipping_receiving/delivery-method-for-carrier>`.

In the :guilabel:`Warehouse` section, tick the :guilabel:`Storage Locations` checkbox to assign
specific locations in the warehouse as loading zones for delivery trucks.

.. _inventory/shipping_receiving/delivery-method-for-carrier:

Delivery method
---------------

Next, assign each delivery vehicle as a :guilabel:`Carrier` by configuring a delivery method.

> To configure delivery methods, go to :menuselection:`Inventory app --> Configuration --> Delivery
> Methods`. Select an existing delivery method, or click :guilabel:`New`.

Reviewer 2: ```suggestion
To configure delivery methods, go to :menuselection:`Inventory app --> Configuration --> Delivery:
Delivery Methods`. Select an existing delivery method, or click :guilabel:`New`.
```

.. seealso::
   :doc:`Configure delivery method <../setup_configuration>`

On the :guilabel:`Delivery Method` form, enter a name for the delivery method. It is recommended to

> use identifying information, such as the vehicle description and license plate number, such as
> `Truck 123-ABCD`.

Reviewer 2: ```suggestion
use identifying information, such as the vehicle description and license plate number (e.g.
`Truck 123-ABCD`).
```

Since the delivery methods are managed internally, set the :guilabel:`Provider` to either
:guilabel:`Fixed Price` or :guilabel:`Based on Rules`. For more information about how shipping
prices are calculated, refer to the :doc:`Delivery method <../setup_configuration>` article.

Next, set a :guilabel:`Delivery Product`, which is the product that shows up as the customer's
:ref:`delivery charge <inventory/shipping_receiving/delivery-product>` on the sales order or
invoice.

Optionally, in the :guilabel:`Availability` tab, set the :guilabel:`Countries`, :guilabel:`States`,
or :guilabel:`Zip Prefixes` to limit the range of local delivery.

.. figure:: dispatch/delivery-method.png
   :alt: Delivery method form.

   Example delivery method, with the :guilabel:`Zip Prefixes` set to San Francisco's zip code.

.. _inventory/shipping_receiving/docks:

Dock locations
--------------

Each loading dock must have a dedicated location. To create or configure dock locations, go to

> :menuselection:`Inventory app --> Configuration --> Locations`.

Reviewer 2: ```suggestion
:menuselection:`Inventory app --> Configuration --> Warehouse Management: Locations`.
```

Author: Hi @ksc-odoo !

According to the [content guidelines](https://www.odoo.com/documentation/16.0/contributing/documentation/content_guidelines.html#lists),  it seems that section names are no longer included when navigating through menu items, and we're now listing only the items themselves. Could you confirm if this is correct?

cc: @samueljlieber

> Click the desired location, which opens the :guilabel:`Location`  form. In the :guilabel:`Additional

Reviewer 2: ```suggestion
Click the desired location, which opens the :guilabel:`Location` form. In the :guilabel:`Additional
```

Information` section, tick the :guilabel:`Is a Dock Location` checkbox.

.. figure:: dispatch/dock-location.png
  :alt: Location configuration.

  Location configuration page with :guilabel:`Is a Dock Location` checkbox ticked.

Build loads
===========

> Once setup is complete, :ref:`assigned orders to a carrier

Reviewer 2: ```suggestion
Once setup is complete, :ref:`assign orders to a carrier
```

<inventory/shipping_receiving/assign-carrier>` and :ref:`group them into batches
<inventory/shipping_receiving/create-batch>`. Then, :ref:`configure the batch form

> <inventory/shipping_receiving/batch-form>` as needed.

Reviewer 2: ```suggestion
<inventory/shipping_receiving/batch-form>`, as needed.
```

> Since this article is about a specific use case, explore details about each picking methods in their

Reviewer 2: ```suggestion
Since this article is about a specific use case, explore details about each picking method in their
```

dedicated articles.

.. seealso::
   - :doc:`../picking_methods/batch`
   - :doc:`Wave picking <../picking_methods/wave>`
   - :doc:`../picking_methods/cluster`

To group products, go to the :menuselection:`Inventory app --> Operations --> Deliveries`, which
reveals a list of outgoing deliveries.

.. _inventory/shipping_receiving/assign-carrier:

Carrier assignment
------------------

> Reveal the :guilabel:`Carrier` column if it is not visible by default, by clicking the

Reviewer 2: ```suggestion
Reveal the *Carrier* column, if it is not visible by default, by clicking the
```

> :icon:`oi-settings-adjust` :guilabel:`(settings)` icon in the to-right corner, and ticking the :guilabel:`Carrier` checkbox.

Reviewer 2: this seems like it exceeds the 100 character limit

.. tip::

>    Other useful columns to enable can be :guilabel:`Zip` code, :guilabel:`Shipping Weight` and

Reviewer 2: ```suggestion
   Other useful columns to enable can be :guilabel:`Zip` code, :guilabel:`Shipping Weight`, and
```

   :guilabel:`Shipping Volume`.

Select the delivery orders for the batch by ticking the checkboxes on the left. Next, click into the
line's :guilabel:`Carrier` fields. In the resulting drop-down menu, choose the desired vehicle's
:ref:`delivery method <inventory/shipping_receiving/delivery-method-for-carrier>`. A
:guilabel:`Confirmation` pop-up window appears, indicating the number of orders being added to the

> batch. Click :guilabel:`Confirm` and the carrier is updated for all the selected records.

Reviewer 2: ```suggestion
batch. Click :guilabel:`Confirm`, and the carrier is updated for all the selected records.
```

.. figure:: dispatch/set-carrier.png
   :alt: Set carrier.

>    The delivery method `Truck 1-MER-001` is set as the :guilabel:`Carrier` for two delivery orders.

Reviewer 2: I could be mistaken, but I'm not sure if GUIs appear in figure captions

.. _inventory/shipping_receiving/create-batch:

Create batch
------------

With the carrier set, begin adding orders to a batch or wave transfer by ticking the checkbox.

.. note::

>   If a delivery order is already assigned to a batch transfer, assigning a batch transfer here will

Reviewer 2: ```suggestion
  If a delivery order is already assigned to a batch transfer, assigning a batch transfer here does
```

  **not** update it.

Then, click the :icon:`fa-cog` :guilabel:`Actions` button, and click either :guilabel:`Add to batch`

> or :guilabel:`Add to wave`. In the pop-up window, ensure :guilabel:`Add to` is set to :guilabel:`a

Reviewer 2: ```suggestion
or :guilabel:`Add to wave`. In the pop-up window, ensure the :guilabel:`Add to` field is set to :guilabel:`a
```

new [batch/wave] transfer`, then click :guilabel:`Confirm`.

.. figure:: dispatch/add-to-wave.png
   :alt: Example wave.

   Delivery orders are selected to be grouped into a wave transfer.

Alternative batch creation method
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> Another place to create batches is by going to the :menuselection:`Inventory` app, and in the

Reviewer 2: ```suggestion
Another place to create batches is by going to the :menuselection:`Inventory app`, and in the
```

:guilabel:`Delivery Orders` card, click the :icon:`fa-ellipsis-v` :guilabel:`(three dots)` icon.
In the resulting drop-down menu, click :guilabel:`Prepare batch`.

.. image:: dispatch/prepare-batch.png
  :alt: Show prepare batch option from the new ... options.

.. note::
   In the :guilabel:`Transport Management` drop-down menu, these other options are used for the

>    following:

Reviewer 2: Where is this located. Some minor elaboration on this would be helpful.

Author: This is the menu that I referred to in the line right before this. I'm not sure if elaboration is necessary here, unless the user cannot see the image.

>    - :guilabel:`Manage Batches`: open list of batches.
>    - :guilabel:`Dock Dispatching`: open weekly calendar view of scheduled batch operations.
>    - :guilabel:`Batches by Route`: kanban view of batches grouped by fulfillment route.
>    - :guilabel:`Calendar`: open hourly calendar view of scheduled operations.
>    - :guilabel:`Statistics`: open pivot table of the batch transfers.

Reviewer 2: ```suggestion
   - :guilabel:`Manage Batches`: open list of batches
   - :guilabel:`Dock Dispatching`: open weekly calendar view of scheduled batch operations
   - :guilabel:`Batches by Route`: kanban view of batches grouped by fulfillment route
   - :guilabel:`Calendar`: open hourly calendar view of scheduled operations
   - :guilabel:`Statistics`: open pivot table of the batch transfers
```

==========================
Dispatch management system
==========================

The *dispatch management system* feature in Odoo is used to plan and build shipments. Key features
include:

- **Load building**: Group products for specific carriers, assign those :doc:`batches
  <../picking_methods/batch>` to loading docks, and manage vehicle assignments based on fleet
  capacity. This ensures the right products are packed into the appropriate trucks for delivery.
- :doc:`Fleet management <../../../../hr/fleet>`: Track and manage the capacity of in-house
  delivery vehicles.

Configuration
=============

To use the dispatch management system, the following setup must be completed:

> #. Install the **Fleet** application.

Reviewer: Suggestion to link to the install doc :)
```suggestion
#. :ref:`Install <general/install>` the **Fleet** application.
```

#. Configure vehicle :ref:`capacity (volume and weight)
   <inventory/shipping_receiving/vehicle-capacity>`.
#. Enter vehicle :doc:`car model(s) <../../../../hr/fleet/models>`.
#. Enable :ref:`necessary features <inventory/shipping_receiving/inventory-features>` in the
   **Inventory** app.
#. Set up :ref:`vehicles as delivery methods
   <inventory/shipping_receiving/delivery-method-for-carrier>`.
#. Create :ref:`dock locations <inventory/shipping_receiving/docks>`.

.. _inventory/shipping_receiving/vehicle-capacity:

Vehicle capacity
----------------

To configure the vehicle capacity, go to :menuselection:`Fleet app --> Configuration -->
Categories`.

Next, click :guilabel:`New` to add a new category, or click into an existing category to modify it.
In the :guilabel:`Name` field, enter the type of vehicle (e.g., `Pick-up truck`, `Van`, or `Cargo
truck`). Then, enter the vehicle capacity in the :guilabel:`Max Weight` (in kilograms) and

> :guilabel:`Volume` (in cubic meters) fields. `

Reviewer: UI field name is "Max Volume" and remove extra backtick :)
```suggestion
:guilabel:`Max Volume` (in cubic meters) fields.
```

> .. note::
>    The units of measurement for vehicle capacity are assigned at the global level in the
>    **Settings** app, in the :guilabel:`Units of Measure` section.
> 
>    .. image:: dispatch/settings.png
>       :alt: Show units of measure settings.
> 
> .. seealso::
>    :ref:`Vehicle category <fleet/categories>`
> 
> .. image:: dispatch/categories.png
>    :alt: Vehicle categories with defined weight and volume.

Reviewer: I suggest swapping the order of these elements, so that the relevant image follows the paragraph, then the note admonition, then the seealso. Otherwise, the seealso seems to break the flow and image loses some context IMO
```suggestion

.. image:: dispatch/categories.png
   :alt: Vehicle categories with defined weight and volume.

.. note::
   The units of measurement for vehicle capacity are assigned at the global level in the
   **Settings** app, in the :guilabel:`Units of Measure` section.

   .. image:: dispatch/settings.png
      :alt: Show units of measure settings.

.. seealso::
   :ref:`Vehicle category <fleet/categories>`

```

Car model
---------

Configuring a vehicle's car model is required when adding vehicles in Odoo. Ensure the correct
*Category* is selected for a car model. Doing so automatically applies weight and volume capacities
to all vehicles of that type.

To configure, navigate to :menuselection:`Fleet app --> Configuration --> Models`.

> .. seealso::
>    :doc:`Create car model <../../../../hr/fleet/models>`
> 
> From the :guilabel:`Models` list, select an existing model, or click :guilabel:`New` in the top-left
> corner to create a new model. Then, set the relevant :guilabel:`Category` field to the relevant
> vehicle category.

Reviewer: Again, suggesting to swap the seealso position to come after the paragraph :)
```suggestion
From the :guilabel:`Models` list, select an existing model, or click :guilabel:`New` in the top-left
corner to create a new model. Then, set the relevant :guilabel:`Category` field to the relevant
vehicle category.

.. seealso::
   :doc:`Create car model <../../../../hr/fleet/models>`

```

.. _inventory/shipping_receiving/inventory-features:

Inventory settings
------------------

Next, go to :menuselection:`Inventory app --> Configuration --> Settings`, and enable the required
features for dispatch management.

In the :guilabel:`Operations` section, tick the :guilabel:`Batch, Wave & Cluster Transfers`
checkbox to prepare batches of orders for delivery.

In the :guilabel:`Shipping` section, tick the :guilabel:`Delivery Methods` and :guilabel:`Dispatch
Management System` checkboxes. Doing so allows specific vehicles to be :ref:`set as carriers
<inventory/shipping_receiving/delivery-method-for-carrier>`.

In the :guilabel:`Warehouse` section, tick the :guilabel:`Storage Locations` checkbox to assign
specific locations in the warehouse as loading zones for delivery trucks.

Then, once all the configurations are complete, be sure to click :guilabel:`Save`.

.. _inventory/shipping_receiving/delivery-method-for-carrier:

Delivery method

> ---------------

Reviewer: Just double checking- it isnt stated anywhere in this section whether or not to publish the newly created/configured delivery method. Is it necessary to state this?

Author: Thanks for being thorough :blush: Publishing the delivery method is necessary if it's available on the website. I don't think it's necessary to mention this, since it's more related to ecommerce stuff and is mentioned on the delivery method doc

:guilabel:`Fixed Price` or :guilabel:`Based on Rules`. For more information about how shipping
prices are calculated, refer to the :doc:`Delivery method <../setup_configuration>` article.

Next, set a :guilabel:`Delivery Product`, which is the product that shows up as the customer's
:ref:`delivery charge <inventory/shipping_receiving/delivery-product>` on the sales order or
invoice.

Optionally, in the :guilabel:`Availability` tab, set the :guilabel:`Countries`, :guilabel:`States`,
or :guilabel:`Zip Prefixes` to limit the range of local delivery.

.. figure:: dispatch/delivery-method.png
   :alt: Delivery method form.

   Example delivery method, with the :guilabel:`Zip Prefixes` set to San Francisco's zip code.

.. _inventory/shipping_receiving/docks:

Dock locations
--------------

Each loading dock must have a dedicated location. To create or configure dock locations, go to
:menuselection:`Inventory app --> Configuration --> Locations`.

Click the desired location, which opens the :guilabel:`Location` form. In the :guilabel:`Additional
Information` section, tick the :guilabel:`Is a Dock Location` checkbox.

> .. figure:: dispatch/dock-location.png
>   :alt: Location configuration.
> 
>   Location configuration page with :guilabel:`Is a Dock Location` checkbox ticked.

Reviewer: Figure attribute alignment :)
```suggestion
.. figure:: dispatch/dock-location.png
   :alt: Location configuration.

   Location configuration page with :guilabel:`Is a Dock Location` checkbox ticked.
```

Build loads
===========

Once setup is complete, :ref:`assign orders to a carrier
<inventory/shipping_receiving/assign-carrier>` and :ref:`group them into batches
<inventory/shipping_receiving/create-batch>`. Then, :ref:`configure the batch form
<inventory/shipping_receiving/batch-form>`, as needed.

> Since this article is about a specific use case, explore details about each picking method in their
> dedicated articles.
> 
> .. seealso::
>    - :doc:`../picking_methods/batch`
>    - :doc:`Wave picking <../picking_methods/wave>`
>    - :doc:`../picking_methods/cluster`
> 
> To group products, go to the :menuselection:`Inventory app --> Operations --> Deliveries`, which
> reveals a list of outgoing deliveries.

Reviewer: Another suggestion to swap seealso position and text, just to avoid breaking the flow of the paragraph
```suggestion
To group products, go to the :menuselection:`Inventory app --> Operations --> Deliveries`, which
reveals a list of outgoing deliveries.

.. seealso::
   Since this article is about a specific use case, explore details about each picking method in
   their dedicated articles.

   - :doc:`../picking_methods/batch`
   - :doc:`Wave picking <../picking_methods/wave>`
   - :doc:`../picking_methods/cluster`

```

.. _inventory/shipping_receiving/assign-carrier:

Carrier assignment
------------------

Reveal the *Carrier* column, if it is not visible by default, by clicking the

> :icon:`oi-settings-adjust` :guilabel:`(settings)` icon in the to-right corner, and ticking the

Reviewer: Missing a "p" ;)
```suggestion
:icon:`oi-settings-adjust` :guilabel:`(settings)` icon in the top-right corner, and ticking the
```

With the carrier set, begin adding orders to a batch or wave transfer by ticking the checkbox.

.. note::
  If a delivery order is already assigned to a batch transfer, assigning a batch transfer here does
  **not** update it.

Then, click the :icon:`fa-cog` :guilabel:`Actions` button, and click either :guilabel:`Add to batch`
or :guilabel:`Add to wave`. In the pop-up window, ensure :guilabel:`Add to` is set to :guilabel:`a
new [batch/wave] transfer`, then click :guilabel:`Confirm`.

.. figure:: dispatch/add-to-wave.png
   :alt: Example wave.

   Delivery orders are selected to be grouped into a wave transfer.

Alternative batch creation method
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Another place to create batches is by going to the :menuselection:`Inventory` app, and in the
:guilabel:`Delivery Orders` card, click the :icon:`fa-ellipsis-v` :guilabel:`(three dots)` icon. In
the resulting drop-down menu, click :guilabel:`Prepare batch`.

.. note::
   The :guilabel:`Transport Management` drop-down menu contains other tools for fleet management:

   - :guilabel:`Manage Batches`: open list of batches
   - :guilabel:`Dock Dispatching`: open weekly calendar view of scheduled batch operations

>    - :guilabel:`Batches by Route`: kanban view of batches grouped by fulfillment route

Reviewer: Please capitalize "Kanban" :)
```suggestion
   - :guilabel:`Batches by Route`: Kanban view of batches grouped by fulfillment route
```

   - :guilabel:`Calendar`: open hourly calendar view of scheduled operations
   - :guilabel:`Statistics`: open pivot table of the batch transfers

> .. image:: dispatch/prepare-batch.png
>   :alt: Show prepare batch option from the new ... options.

Reviewer: Image alt alignment and small alt tag change to be more descriptive. 

Also, I'd like to see this image cropped to just the delivery orders card, IMO there is no need to show the whole UI here :)

<img width="267" alt="Screenshot 2024-10-22 at 11 30 55 AM" src="https://github.com/user-attachments/assets/2b4ebb32-22e5-42c3-aab9-087b8613d790">

```suggestion
.. image:: dispatch/prepare-batch.png
   :alt: Show prepare batch option from the Transport Management drop-down menu.
```
